src/components/ClientCard/styles.js

```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+
+const style = (Component) => styled(Component)`
+  * {
+    font-family: Lato;
```

Comment:
We are already setting the font family in the Layout component.

Reply:
Fixed, removing at commit 21d4a4b